docs/production-deployment/cloud/nexus/latency-availability.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: latency-availability
+slug: /cloud/nexus/latency-availability
+title: Latency and Availability - Temporal Nexus
+description: Learn about Nexus latency and availability
+sidebar_label: Latency and Availability
+tags:
+  - Temporal Cloud
+  - Nexus
+keywords:
+  - Temporal Nexus
+  - Nexus
+  - latency
+  - availability
+  - SLO
+  - SLA
+  - error rate
+  - service level
+  - service level objective
+  - service level agreement
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview).
+Learn why you should use Nexus in the [evaluation guide](/evaluate/nexus).
+
+:::
+
+Nexus has the same [latency SLOs](/cloud/service-availability#latency) and [availability SLAs](/cloud/sla) for Nexus requests (commands, polling, responses) between Workers and Temporal Cloud.
```

Comment:
Nexus has the same ... for Nexus is confusing to me.

Reply:
fixed